guides/common/modules/con_overview-of-load-balancing-in-project.adoc

```diff
@@ -4,6 +4,5 @@
 = Overview of load balancing in {Project}
 
 [role="_abstract"]
-You can configure your {Project} environment to use a load balancer to distribute host requests and network load across multiple {SmartProxyServers}.
-This results in an improved performance on {SmartProxyServers} and improved performance and stability for host connections to {Project}.
+You can configure your {Project} environment to use a load balancer to distribute host requests and network load across multiple {SmartProxyServers}. This results in an improved performance on {SmartProxyServers} and improved performance and stability for host connections to {Project}.
 In a load-balanced setup, {SmartProxy} functionality supported for load balancing continues to work as expected when one {SmartProxyServer} is down for planned or unplanned maintenance.
```

Comment:
Same comment as below. I think the rule is not detecting the second sentence because it is followed immediately by a line break.